hudi-common/src/main/java/org/apache/hudi/common/table/HoodieTableMetaClient.java

```diff
@@ -390,7 +389,9 @@ public HoodieStorage getStorage() {
           consistencyGuardConfig)
           : new NoOpConsistencyGuard();
 
-      storage = getStorageWithWrapperFS(
+      storage = (HoodieStorage) ReflectionUtils.loadClass("org.apache.hudi.storage.hadoop.HoodieHadoopStorage",
```

Comment:
Define a static final String for `"org.apache.hudi.storage.hadoop.HoodieHadoopStorage"`?